src/main/java/com/heeverse/ticket_order/service/reader/firstclass/ResultMap.java

```diff
@@ -0,0 +1,44 @@
+package com.heeverse.ticket_order.service.reader.firstclass;
+
+import com.heeverse.ticket_order.domain.dto.persistence.AggregateInsertMapperDto;
+import com.heeverse.ticket_order.service.reader.producer.TaskMessage;
+
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.atomic.LongAdder;
+
+import static com.heeverse.ticket_order.domain.dto.persistence.AggregateSelectMapperDto.SimpleResponse;
+
+/**
+ * @author gutenlee
+ * @since 2023/10/28
+ */
+public class ResultMap {
+
+    private final ConcurrentHashMap<String, LongAdder> resultMap;
+
+    public ResultMap() {
+        this.resultMap = new ConcurrentHashMap<>();
+    }
+
+    public void add(Map.Entry<String, Long> entry) {
+        resultMap.computeIfAbsent(entry.getKey(), r -> getInit());
+        resultMap.computeIfPresent(entry.getKey(), (key, a) -> {
+            a.add(entry.getValue());
+            return a;
+        });
```

Comment:
`compute`라는 메서드로 단일연산으로 만들어주는게 더 좋아보입니다.

`computeIfAbsent` 후 `computeIfPresent`를 호출하기 때문에 동시성 이슈가 발생할일은 거의 없어보이는데요, 원칙적으로 두가지 이상의 연산을 락 없이 진행할 때에는 동시성 이슈가 발생할 수 있습니다.

concurrent hash map이 동시성을 제어하는 방법은 두가지 이상의 연산을 하나로 묶을 수 있는 기능을 제공하기 때문입니다. 그런데 이런 연산을 여러 스레드레서 각각 여러번 호출하면 당연하게도 동시성 이슈가 발생할 수 있습니다. 그러므로 아예 하나의 원자적인 연산을 만들어서 처리해주면 동시성 관련 고민을 안해도 될 것으로 보이네요~
```suggestion
        resultMap.compute(entry.getKey(), r -> r == null ? getInit() : r.add(entry.getValue());
```